src/addr_families.c

```diff
@@ -826,12 +831,75 @@ WOLFSENTRY_API wolfsentry_errcode_t wolfsentry_addr_family_ntop(
 
 #endif /* WOLFSENTRY_PROTOCOL_NAMES */
 
+static wolfsentry_errcode_t wolfsentry_addr_family_max_addr_bits_1(
+    wolfsentry_addr_family_t family,
+    wolfsentry_addr_bits_t *bits)
+{
+    switch(family) {
+    case WOLFSENTRY_AF_INET:
+        { *bits = 32; WOLFSENTRY_RETURN_OK; }
```

Comment:
What are these max bits for? The IP address?

Reply:
yup that's just the usual "a complete IPv4 address is 32 bits".  wolfSentry has to implement some of these sorts of lookups internally for portability.